### ISSUE TITLE
Problem with Anton Lesser name

### ISSUE THREAD

**Issue by muttetjeff** [pdulvp/jellyfin-qnap#52]
Hello, I installed Jellyfin on a Qnap TS-253D and everything works fine. Except that I am unable to add the name of Anton Lesser (actor who plays in the series Endeavour) to the list of actors. When I click on 'save' it never appears in the bottom list. I tried several times and it doesn't work. This is the only artist name that I am unable to add. Really weird...

**Comment by RedNo7** [comment]
This sounds more like a Jellfin core issue (rather than a QNAP packaging issue).

**Comment by Migamix** [comment]
yes, this is 100% a jellyfin core issue and not an app compatibility issue, please remove/close this as an issue for this port and pursue the issue on jellyfin main forums or their GIT
